src/components/tables/DataTable.js

```diff
@@ -75,10 +76,21 @@ export default {
     value: {
       type: Array,
       default: () => []
+    },
+    totalItems: {
+      type: Number,
+      default: null
+    },
+    loading: {
+      type: Boolean,
+      default: false
     }
   },
 
   computed: {
+    length () {
```

Comment:
can we give this a more descriptive name? seeing ```this.value.length``` and ```this.length``` throughout was a bit confusing.